--- baseline.cpp	2022-03-28 14:59:23.985945207 +0200
+++ baseline.cpp	2022-03-28 15:05:56.047805308 +0200
@@ -780,8 +780,12 @@
     evmc_revision rev, const evmc_message* msg, const uint8_t* code, size_t code_size) noexcept
 {
     auto vm = static_cast<VM*>(c_vm);
-    const auto jumpdest_map = analyze(code, code_size);
-    auto state = std::make_unique<ExecutionState>(*msg, rev, *host, ctx, code, code_size);
-    return execute(*vm, *state, jumpdest_map);
+    evmc_result result = evmc::make_result(EVMC_REJECTED, 0, nullptr, 0);
+    for (unsigned int run_id = 0; run_id < 1000000; ++run_id) {
+        const auto jumpdest_map = analyze(code, code_size);
+        auto state = std::make_unique<ExecutionState>(*msg, rev, *host, ctx, code, code_size);
+        result = execute(*vm, *state, jumpdest_map);
+    }
+    return result;
 }
 }  // namespace evmone::baseline
